cluster/local/integration_tests.sh

```diff
@@ -58,16 +58,26 @@ if [ "$skipcleanup" != true ]; then
     trap cleanup EXIT
 fi
 
-echo_step "creating k8s cluster using kind"
-"${KIND}" create cluster --name="${K8S_CLUSTER}"
+readonly DEFAULT_KIND_CONFIG="kind: Cluster
+apiVersion: kind.x-k8s.io/v1alpha4
+"
+if $skipcleanup && $(echo $("${KIND}" get clusters) | grep -wq "${K8S_CLUSTER}"); then
+    echo "existing kind cluster: ${K8S_CLUSTER}"
+else
+    echo_step "creating k8s cluster using kind with the following config:"
+    kind_config="${KIND_CONFIG:-$DEFAULT_KIND_CONFIG}"
+    echo "${kind_config}"
+    echo "${kind_config}" | "${KIND}" create cluster --name="${K8S_CLUSTER}" --config=-
+fi
 
 # tag crossplane image and load it to kind cluster
 docker tag "${BUILD_IMAGE}" "${CROSSPLANE_IMAGE}"
 "${KIND}" load docker-image "${CROSSPLANE_IMAGE}" --name="${K8S_CLUSTER}"
 
 echo_step "installing helm package(s) into \"${CROSSPLANE_NAMESPACE}\" namespace"
-"${KUBECTL}" create ns "${CROSSPLANE_NAMESPACE}"
-"${HELM3}" install "${PROJECT_NAME}" --namespace "${CROSSPLANE_NAMESPACE}" "${projectdir}/cluster/charts/${PROJECT_NAME}" --set replicas=2,rbacManager.replicas=2,image.pullPolicy=Never,imagePullSecrets=''
+"${KUBECTL}" create ns "${CROSSPLANE_NAMESPACE}" || $skipcleanup
+$skipcleanup && "${HELM3}" uninstall "${PROJECT_NAME}" --namespace "${CROSSPLANE_NAMESPACE}" || true
+"${HELM3}" upgrade --install "${PROJECT_NAME}" --namespace "${CROSSPLANE_NAMESPACE}" "${projectdir}/cluster/charts/${PROJECT_NAME}" --set replicas=2,args={'--debug'},rbacManager.replicas=2,rbacManager.args={'--debug'},image.pullPolicy=Never,imagePullSecrets=''
 
 echo_step "waiting for deployment ${PROJECT_NAME} rollout to finish"
```

Comment:
It looks like stale code, currently helm supports `--wait` or `--atomic` option for this purpose:
>--wait                                       if set, will wait until all Pods, PVCs, Services, and minimum number of Pods of a Deployment, StatefulSet, or ReplicaSet are in a ready state before marking the release as successful. It will wait for as long as --timeout

the code of feature:
https://github.com/helm/helm/blob/3974136b6eba92f17a821aac52a80b59a9fa1202/pkg/kube/wait.go#L45-L62

However it's up to you

Reply:
I haven't added it because of backward compatibility reasons